.gitattributes

```diff
@@ -0,0 +1,16 @@
+* text=auto
```

Comment:
The problem is that you know what line endings files should be created as but unfortunately not everyone does. By keeping the .gitattribute file in the repo, we can ensure that any file created uses the proper line endings. The java client didn't have this and every other file had different line endings which made things frustrating at times.